.github/workflows/bazel.yml

```diff
@@ -32,3 +32,26 @@ jobs:
         run: bazel --output_user_root=~/.bazel/cache build //...
       - name: Test All
         run: bazel --output_user_root=~/.bazel/cache test //...
+
+  # iOS is 10x expensive to run on GitHub machines, so only run if we know something else passed
+  # The steps are unfortunately duplicated because github actions requires 2 jobs for a dependency
+  build-macos:
+    needs: build
```

Comment:
how this looks

![image](https://github.com/user-attachments/assets/bd31935a-f938-404d-baf6-3a10e2451f4a)